src/main/java/com/iota/iri/controllers/MilestoneViewModel.java

```diff
@@ -252,15 +250,7 @@ private static void cachePut(Tangle tangle, MilestoneViewModel milestoneViewMode
      * @return Cache found.
      */
     private static Cache<Indexable, MilestoneViewModel> getCache(Tangle tangle) {
-        CacheManager cacheManager = tangle.getCacheManager();
-        if (cacheManager == null) {
-            return null;
-        }
-        Cache<Indexable, MilestoneViewModel> cache = cacheManager.lookup(MilestoneViewModel.class);
-        if (cache != null) {
-            return cache;
-        }
-        return tangle.getCacheManager().add(MilestoneViewModel.class, new MilestoneCacheConfiguration());
+        return tangle.getCache(MilestoneViewModel.class);
```

Comment:
remember to fix this as well

Reply:
Nothing to fix here since we are using `initializeCaches`